great_expectations/expectations/core/expect_multicolumn_sum_to_equal.py

```diff
@@ -227,6 +232,49 @@ def schema_extra(schema: Dict[str, Any], model: Type[ExpectMulticolumnSumToEqual
                 }
             )
 
+    @classmethod
+    def _prescriptive_template(
+        cls,
+        renderer_configuration: RendererConfiguration,
+    ) -> RendererConfiguration:
+        add_param_args: AddParamArgs = (
+            ("column_list", RendererValueType.ARRAY),
+            ("sum_total", RendererValueType.NUMBER),
+            ("mostly", RendererValueType.NUMBER),
+            ("ignore_row_if", RendererValueType.STRING),
+        )
+        for name, param_type in add_param_args:
+            renderer_configuration.add_param(name=name, param_type=param_type)
+
+        params = renderer_configuration.params
+
+        template_str = ""
+        if params.column_list:
+            array_param_name = "column_list"
+            param_prefix = "column_list_"
+            renderer_configuration = cls._add_array_params(
+                array_param_name=array_param_name,
+                param_prefix=param_prefix,
+                renderer_configuration=renderer_configuration,
+            )
+            template_str += "Sum across columns " + cls._get_array_string(
+                array_param_name=array_param_name,
+                param_prefix=param_prefix,
+                renderer_configuration=renderer_configuration,
+            )
+
+        if params.mostly and params.mostly.value < 1.0:
+            renderer_configuration = cls._add_mostly_pct_param(
+                renderer_configuration=renderer_configuration
+            )
+            template_str += " must be $sum_total, at least $mostly_pct % of the time."
+        else:
+            template_str += " must be $sum_total."
```

Comment:
`non-blocking`
You could accumulate these items in a list and then `.join()` them at the end.
String operations in python are actually relatively fast though, so there might not actually be much benefit to this. 🤷 
```python
template_parts: list[str] = []
if params.column_list:
  array_param_name = "column_list"
  param_prefix = "column_list_"
  renderer_configuration = cls._add_array_params(
    array_param_name=array_param_name,
    param_prefix=param_prefix,
    renderer_configuration=renderer_configuration,
  )
  template_parts += "Sum across columns "
  template_parts += cls._get_array_string(
    array_param_name=array_param_name,
    param_prefix=param_prefix,
    renderer_configuration=renderer_configuration,
  )

...

  renderer_configuration.template_str = "".join(template_parts)
  return renderer_configuration
```